extensions/purelymail/src/check-account-credit.tsx

```diff
@@ -0,0 +1,18 @@
+import { Clipboard, showToast, Toast, updateCommandMetadata } from "@raycast/api";
+import { getAccountCredit } from "./utils/api";
+import { Response } from "./utils/types";
+import { showFailureToast } from "@raycast/utils";
+
+export default async function CheckAccountCredit() {
+  const response: Response = await getAccountCredit();
+
+  if (response.type === "error") {
+    await showFailureToast(response.message, { title: response.code });
+  } else {
+    const { credit } = response.result;
+    const roundedCredit = parseFloat(credit as string).toFixed(2);
+    await Clipboard.copy(roundedCredit);
```

Comment:
Does it make sense to copy to the clipboard? I would maybe not do that since that's unexpected behavior. 😊

Reply:
I can follow that logic.
Updated so that:

1. Credit is shown as formatted currency since that is how it is shown in Purelymail Billing AND
2. No longer copied to Clipboard.